docs/src/components/home/ProductSuite.tsx

```diff
@@ -56,11 +56,10 @@ const ProductSuite = () => {
                 overline="Products"
                 title={
                   <Typography variant="h2" sx={{ my: 1 }}>
-                    Extensive library of components, ready for{' '}
-                    <GradientText>production</GradientText>
+                    Every component you need is <GradientText>ready to ship</GradientText> today
                   </Typography>
                 }
-                description="A suite of products integrated to make your life easier when it comes to setting up design systems."
+                description="Empower your team to build at an accelerated pace without sacrificing flexibility or control."
```

Comment:
> Empower

Isn't that on the buzzword blacklist?

Reply:
I don't hate it but I'm not married to it either. We could drop that first clause entirely.

```suggestion
                description="Build at an accelerated pace without sacrificing flexibility or control."
```